gensim/models/fasttext_inner.pyx

```diff
@@ -317,7 +317,8 @@ def train_batch_sg(model, sentences, alpha, _work, _l1):
                 continue
             indexes[effective_words] = word.index
 
-            subwords = [model.wv.ngrams[subword_i] for subword_i in model.wv.ngrams_word[model.wv.index2word[word.index]]]
+            subwords = [model.wv.hash2index[ft_hash(subword) % model.bucket]
```

Comment:
Please use only hanging indents (no vertical).

Reply:
That is, remove the newline and keep the comprehension on a single line?

Reply:
single line (if line <= 120 characters) or something like
```python
subwords = [
    ....
    ....
]
```